hack/jenkins/common.sh

```diff
@@ -441,6 +441,11 @@ if [ -z "${EXTERNAL}" ]; then
   gsutil -qm cp "${HTML_OUT}" "gs://${JOB_GCS_BUCKET}.html" || true
   echo ">> uploading ${SUMMARY_OUT}"
   gsutil -qm cp "${SUMMARY_OUT}" "gs://${JOB_GCS_BUCKET}_summary.json" || true
+  if [[ "${MINIKUBE_LOCATION}" == "master" ]]; then
+    ./test-flake-chart/upload_tests.sh "${SUMMARY_OUT}"
+  elif [[ "${JOB_NAME}" == "Docker_Linux" ]]; then
```

Comment:
This elif is a temporary solution to get this flake rate stuff stood up. In the future, all environments should be reporting their flakes, but that requires figuring out how to synchronize after all Jenkins integration tests have finished, which we don't have a working model for. Future PR!